.travis.yml

```diff
@@ -2,15 +2,32 @@ language: julia
 os:
   - linux
   - osx
+  - windows
+arch:
+  - x64
+  - x86
+  - arm64
 julia:
-  - 0.7
-  - 1.0
-  - 1.1
+  - 1.3
   - nightly
 matrix:
   allow_failures:
   - julia: nightly
+  exclude:
+    - os: osx
+      arch: x86
+    - os: osx
+      arch: arm64
+    - os: windows
+      arch: arm64
+    - julia: nightly
+      arch: arm64
 notifications:
     email: false
+
+script:
+ - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
+ - julia -e 'using Pkg; pkg"add https://github.com/staticfloat/Cairo.jl#sf/jll_pkg"; Pkg.build(verbose=true); Pkg.test(coverage=true)'
```

Comment:
This entire section can now be deleted, right?

Reply:
I think so, let's see how it goes without it